docs/source/overview/installing-caliper.md

```diff
@@ -0,0 +1,392 @@
+
+```sh
+user@ubuntu:~/caliper-benchmarks$ npx caliper --help
+caliper <command>
+
+Commands:
+  caliper.js bind [options]       Bind Caliper to a specific SUT and its SDK version
+  caliper.js launch <subcommand>  Launch a Caliper process either in a manager or worker role.
+  caliper.js unbind [options]     Unbind Caliper from a previously bound SUT and its SDK version
+  caliper.js completion           generate completion script
+
+Options:
+  --help, -h  Show usage information  [boolean]
+  --version   Show version information  [boolean]
+
+Examples:
+  caliper bind
+  caliper unbind
+  caliper launch manager
+  caliper launch worker
+
+For more information on Hyperledger Caliper: https://hyperledger.github.io/caliper/
+```
+You can also request the help page of a specific command, as demonstrated by the next subsections.
+
+!!! note
+
+    *the command options can be set either through the command line, or from various other sources supported by the [configuration mechanism](https://hyperledger.github.io/caliper/v0.6.0/reference/runtime-config/) of Caliper. This flexibility makes it easy to embed the CLI in different environments.*
+
+### The bind command
+
+Acquiring Caliper is as easy as installing a single NPM package, or pulling a single Docker image. However, this single point of install necessitates an additional step of telling Caliper which platform to target and which platform SDK version to use. This step is called binding, provided by the `bind` CLI command.
+
+To have a look at the help page of the command, execute:
+```sh
+user@ubuntu:~/caliper-benchmarks$ npx caliper bind --help
+Usage:
+  caliper bind --caliper-bind-sut fabric:fabric-gateway --caliper-bind-cwd ./ --caliper-bind-args="-g"
+
+Options:
+  --help, -h           Show usage information  [boolean]
+  --version            Show version information  [boolean]
+  --caliper-bind-sut   The name and version of the platform and its SDK to bind to  [string]
+  --caliper-bind-cwd   The working directory for performing the SDK install  [string]
+  --caliper-bind-args  Additional arguments to pass to "npm install". Use the "=" notation when setting this parameter  [string]
+  --caliper-bind-file  Yaml file to override default (supported) package versions when binding an SDK  [string]
+```
+The binding step technically consists of an extra `npm install` call with the appropriate packages and install settings, fully managed by the CLI. The following parameters can be set for the command:
+
+- **SUT/platform name and SDK version**: specifies the name of the target platform and the SDK version to install, e.g., `fAbric:2.2`
+- **Working directory**: the directory from which the `npm install` command must be performed. Defaults to the current working directory
+- **User arguments**: additional arguments to pass to `npm install`, e.g., `--save`
+
+The following SUT name and SDK version combinations are supported:
+- **besu**: `1.3.2`, `1.3`, `1.4`

```

Comment:
this section is not formatted correctly the sut names are not on separate lines with bullets